### PR TITLE
Rename SKWebApi to CopilotChatApi

### PR DESCRIPTION
**Pull request by adrianwyatt**
### Motivation and Context
"SKWebApi" is not descriptive enough for intention of the application.

### Description
Renamed SKWebApi to CopilotChatApi everywhere.

### CODE REVIEW

File: dotnet/SK-dotnet.sln
Comment (glahaye) [inline dotnet/SK-dotnet.sln:1]: The goal for the backend is to be SK as a service, not just the backend behind the chat app.

The rename here actually goes away from that and won't sit well when we have SKaaS deployed internally to support various apps, and even less when users will be able to deploy the backend to Azure or a container to support their apps!

File: samples/apps/copilot-chat-app/WebApp/yarn.lock
Comment (craigomatic) [inline samples/apps/copilot-chat-app/WebApp/yarn.lock:1]: It's a little strange to see many changes in this file, given everything else in the repo is a rename maybe this change isn't needed?

Comment (adrianwyatt) [reply samples/apps/copilot-chat-app/WebApp/yarn.lock:1]: Not sure what happened here - re-copied from 'main'.

File: dotnet/SK-dotnet.sln
Comment (adrianwyatt) [reply dotnet/SK-dotnet.sln:1]: Let's keep it in the CopilotChat naming paradigm while it is located in samples/apps/copilot-chat-app. When we're ready to consider the webapi as sufficiently generalized, let's discuss moving it out of the copilot chat sample directory and consider renaming again then.

**Pull request merged**